src/main/kotlin/com/hibob/academy/resource/PetResource.kt

```diff
@@ -0,0 +1,31 @@
+package com.hibob.academy.resource
+
+
+
+
+import com.hibob.academy.service.PetService
+import jakarta.ws.rs.Consumes
+import jakarta.ws.rs.GET
+import jakarta.ws.rs.POST
+import jakarta.ws.rs.PUT
+import jakarta.ws.rs.Path
+import jakarta.ws.rs.PathParam
+import jakarta.ws.rs.Produces
+import jakarta.ws.rs.core.MediaType
+import jakarta.ws.rs.core.Response
+import org.springframework.stereotype.Controller
+
+@Controller
+@Path("/api/matan/sabag/pets/")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+class PetResource (private val petService: PetService) {
+
+
+
+    @GET
+    @Path("countPetsByType/{companyId}")
```

Comment:
should be   @Path("count-by-type/company/{companyId}")
 you can read about rest conventions here
 https://restfulapi.net/resource-naming/